doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -301,8 +301,20 @@ set with::
 SPDK Usage
 ==================
 
-If you want to use SPDK driver for NVME SSD, you need to specify NVMe serial
-number here with "spdk:" prefix for ``bluestore_block_path``.
+If you want to use SPDK driver for NVME SSD, you need to ready your system.
+Please refer to `SPDK document`__ for more detailes.
+
+.. __: http://www.spdk.io/doc/getting_started.html#getting_started_examples
+
+NVME device must be unbound from the native kernel drivers. SPDK offers a
```

Comment:
i think this instruction has been covered by http://www.spdk.io/doc/getting_started.html#getting_started_examples .

Reply:
Will update